src/Elasticsearch/Metadata/Document/Factory/AttributeDocumentMetadataFactory.php

```diff
@@ -1,74 +0,0 @@
-<?php
-
-/*
- * This file is part of the API Platform project.
- *
- * (c) Kévin Dunglas <dunglas@gmail.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-declare(strict_types=1);
-
-namespace ApiPlatform\Elasticsearch\Metadata\Document\Factory;
-
-use ApiPlatform\Elasticsearch\Exception\IndexNotFoundException;
-use ApiPlatform\Elasticsearch\Metadata\Document\DocumentMetadata;
-use ApiPlatform\Metadata\Resource\Factory\ResourceMetadataCollectionFactoryInterface;
-
-/**
- * Creates document's metadata using the attribute configuration.
- *
- * @experimental
- *
- * @author Baptiste Meyer <baptiste.meyer@gmail.com>
- */
-final class AttributeDocumentMetadataFactory implements DocumentMetadataFactoryInterface
```

Comment:
same for these we should keep them but deprecate their usage. I know as they're experimental we could remove them as-is but I'm afraid they're already widely used.